src/libraries/Microsoft.PowerFx.Core/Public/Values/ErrorValue.cs

```diff
@@ -68,5 +68,16 @@ private static IEnumerable<ExpressionError> CombineErrors(IEnumerable<ErrorValue
                 }
             }
         }
+
+        public override string ToExpression()
+        {
+            // !JYL!
+            /* Few questions on this:
+             * Should this be serialized?
```

Comment:
>Should this be serialized? [](http://example.com/codeflow?start=15&length=26)

I don't see why we wouldn't

Reply:
agreed, yes, we should serialize
We have an Error() function we can use.  Can you ask Carlos for details - he'll want to review this.